src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -46,6 +53,10 @@ public boolean isExit() {
         return exit;
     }
 
+    public boolean isSwitchTab() {
+        return !tabType.equals("default");
+    }
```

Comment:
all these "default" are like magic numbers. It's better to have a static final variable DEFAULT_TAB to represent them

Reply:
Resolved